<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <widget class="QDialogButtonBox" name="SettingsButtonBox">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>460</y>
     <width>231</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTabWidget" name="SettingsTabs">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>581</width>
     <height>441</height>
    </rect>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Triangular</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideRight</enum>
   </property>
   <property name="documentMode">
    <bool>true</bool>
   </property>
   <property name="tabsClosable">
    <bool>false</bool>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="Appearance">
    <attribute name="title">
     <string>Appearance</string>
    </attribute>
    <widget class="QLabel" name="FontLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>131</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Choose font</string>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
    <widget class="QComboBox" name="FontComboBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>211</width>
       <height>42</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Helvetica</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Times</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Cantarell</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QWidget" name="Behavior">
    <attribute name="title">
     <string>Behaviour</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>SettingsButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SettingsButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
